content/md/support/manage-support-access.md

```diff
@@ -36,18 +36,52 @@ Note that only the latest issues are displayed on the project console.
 
 Go to the `User Access` tab to see the list of contacts having a support access.
 
+**Add access to an existing contact of your company:**
```

Comment:
Can you put those actions as a secondary level? They will appear in the left menu, and it will be consistent with other access, as defined above ^^.